Mlem/Views/Tabs/Settings/Components/Views/General/GeneralSettingsView.swift

```diff
@@ -113,6 +114,19 @@ struct GeneralSettingsView: View {
             } footer: {
                 Text("Optimizes performance for your internet speed. You may need to restart the app for all optimizations to take effect.")
             }
+            
+            Section {
+                SelectableSettingsItem(
+                    settingIconSystemName: Icons.appLockSettings,
+                    settingName: "App Lock",
+                    currentValue: $appLock,
+                    options: AppLock.allCases
+                )
+            } header: {
+                Text("Privacy")
+            } footer: {
+                Text("Locks your app with Face ID")
+            }
```

Comment:
I think we should use a toggle here until more options are added

```suggestion
                SwitchableSettingsItem(
                    settingPictureSystemName: Icons.appLockSettings,
                    settingName: "Lock with Face ID",
                    isTicked: Binding(
                        get: { appLock == .instant },
                        set: { selected in
                            appLock = selected ? .instant : .disabled
                        }
                    )
                )
            } header: {
                Text("Privacy")
            }
```